UpdatedSyntheticDataset/SyntheticDataset2/ElementsCreator/circle.py

```diff
@@ -0,0 +1,45 @@
+from PIL import ImageDraw, Image
+
+
+class Circle(object):
+
+    def __init__(self, radius, color, midpoint, image):
+        """
+        Initialize a Circle shape
+
+        :param radius: radius in pixels
+        :type radius: int
+        :param color: color of shape - RGB
+        :type color: 3-tuple
+        :param midpoint: midpoint where shape will be overlayed on image
+        :type midpoint: 2-tuple xy pixel coordinates
+        :param image: image for shape to be overlayed on
+        :type image: PIL image
+
+        :param coordinates: drawing coordinates for the shape
```

Comment:
Is this supposed to be in the method declaration? If not, then you do not need the parameter reference here.